docs/EN_US/ECLLanguageReference/ECLR_mods/BltInFunc-BUILD.xml

```diff
@@ -807,6 +808,78 @@ END;
 #WORKUNIT('name','Ppass.FilterDsLib')
 BUILD(FilterDsLib1);
 </programlisting>
+  </sect2>
+
+  <sect2 id="IDXCOMP2" role="brk">
+    <title>Indexes and Compression</title>
+
+    <para><emphasis role="bold">Compression Options:</emphasis></para>
+
+    <informaltable colsep="1" frame="all" rowsep="1">
+      <tgroup cols="2">
+        <colspec align="left" colwidth="122.40pt" />
+
+        <colspec />
+
+        <tbody>
+          <row>
+            <entry><emphasis role="bold">LZW</emphasis></entry>
+
+            <entry>The default compression. It is a variant of the
+            Lempel-Ziv-Welch algorithm. It remains the default for backward
+            compatibility.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">ROW</emphasis></entry>
+
+            <entry>Compresses index entries based on differences between rows
+            (<emphasis role="bold">for use with fixed-length records
+            only</emphasis>). It typically does not compress as well as LZW,
+            but takes up less space in memory because the rows are expanded on
+            demand.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold">'inplace'</emphasis></entry>
+
+            <entry>Causes the index to be built using the inplace compression
+            format. The payload defaults to using lz4 compression.</entry>
+          </row>
+
+          <row>
+            <entry><emphasis role="bold"><emphasis
+            role="bold">'inplace:lz4hc'</emphasis> </emphasis></entry>
+
+            <entry>Causes inplace compression on the key fields and lz4hc
+            compression on the payload. The resulting index can be smaller
+            than using lx4.</entry>
+          </row>
+        </tbody>
+      </tgroup>
+    </informaltable>
+
+    <para>The inplace index compression format (introduced in version 9.2.0)
```

Comment:
Suggest some edits:

  <para>The inplace index compression format (introduced in version 9.2.0)
    improves compression of keyed fields and allows them to be
    searched without decompression. The original index compression
    implementation decompresses the rows when they are read from disk.

    <para>Because the keyed fields can be searched without decompression more
    branch nodes fit into memory which can improve search performance. The lz4
    compression used for the payload is significantly faster at decompressing
    leaf pages than the previous LZW compression.</para>

Because i) LZW isn't used for branch nodes ii) the keyed field compression is used in branch and leaf nodes.